_posts/2016-02-04-meet-the-hoodies-offline.md

```diff
@@ -0,0 +1,54 @@
+---
+
+title: Meet The Hoodies
+author: Gregor
+layout: blog-post
+comments: true
+
+---
+
+# Meet the Hoodies: Offline Edition
+
+Maybe you’ve heard about Hoodie at a conference, or read about on Twitter, or through our initiatives like Offline First and noBackend and want to learn more, or you just want to get started with contributing to a meaningful Open Source project. Either way: **Meet the Hoodies is for you**!
+
+## Let’s meet the Hoodies!
+
+We would love to come visit your city and organise a fun workshop for new and experienced open source contributors, to give an introduction to Hoodie and above all: to meet you in person.
+
+While we love to see the Hoodie community grow, meeting you in person is what makes all the difference. These Github avatars are never going to be the same once you meet the friendly humans behind it – That goes both ways :)
+
+The setup is simple: we maintain a [list of great starter issues](http://go.hood.ie/hoodie-starter-issues) to make it as simple and fun as possible to start contributing to Hoodie. Attendees can choose any issue to work and and get all support you need from Hoodie hosts.
```

Comment:
'Attendees can choose any issue to work' should be 'Attendees can choose any issue to work on'